src/DIRAC/Resources/Computing/PoolComputingElement.py

```diff
@@ -263,3 +251,7 @@ def getDescription(self):
     ceDictList.append(dict(ceDict))
 
     return S_OK(ceDictList)
+
+  def shutdown(self):
+    self.pPool.shutdown()
```

Comment:
From what I understand, the `future.shutdown` will wait until all the processes are done before returning a value.
Maybe you could add a comment about this?